docs/operations/observability.md

```diff
@@ -24,22 +24,26 @@ The Loki Distributors expose the following metrics:
-| `loki_ingester_chunk_size_bytes`          | Histogram   | Distribution of chunk sizes when flushed.                                                   |
-| `loki_ingester_chunk_stored_bytes_total`  | Counter     | Total bytes stored in chunks per tenant.                                                    |
-| `loki_ingester_chunks_created_total`      | Counter     | The total number of chunks created in the ingester.                                         |
-| `loki_ingester_chunks_flushed_total`      | Counter     | The total number of chunks flushed by the ingester.                                         |
-| `loki_ingester_chunks_stored_total`       | Counter     | Total stored chunks per tenant.                                                             |
-| `loki_ingester_received_chunks`           | Counter     | The total number of chunks sent by this ingester whilst joining during the handoff process. |
-| `loki_ingester_samples_per_chunk`         | Histogram   | The number of samples in a chunk.                                                           |
-| `loki_ingester_sent_chunks`               | Counter     | The total number of chunks sent by this ingester whilst leaving during the handoff process. |
-| `loki_ingester_streams_created_total`     | Counter     | The total number of streams created per tenant.                                             |
-| `loki_ingester_streams_removed_total`     | Counter     | The total number of streams removed per tenant.                                             |
+| Metric Name                                  | Metric Type | Description                                                                                 |
+| -------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------- |
+| `cortex_ingester_flush_queue_length`         | Gauge       | The total number of series pending in the flush queue.                                      |
+| `cortex_chunk_store_index_entries_per_chunk` | Histogram   | Number of index entries written to storage per chunk.                                       |
+| `loki_ingester_memory_chunks`                | Gauge       | The total number of chunks in memory.                                                       |
+| `loki_ingester_memory_streams`               | Gauge       | The total number of streams in memory.                                                      |
+| `loki_ingester_chunk_age_seconds`            | Histogram   | Distribution of chunk ages when flushed.                                                    |
+| `loki_ingester_chunk_encode_time_seconds`    | Histogram   | Distribution of chunk encode times.                                                         |
+| `loki_ingester_chunk_entries`                | Histogram   | Distribution of entries per-chunk when flushed.                                             |
```

Comment:
is this the length of the entries?

Reply:
i'm not entirely sure what an "entry" is here

Reply:
ok it looks like entries = log lines, wonder if we should make that clearer in the description

Reply:
Agreed.  Updated the description to include the word "lines" instead of "entries" to ease confusion.